integrationTests/frontend/wallet/txInterception_test.go

```diff
@@ -28,6 +28,8 @@ func TestInterceptedTxFromFrontendGeneratedParamsWithoutData(t *testing.T) {
 }
 
 func TestInterceptedTxFromFrontendGeneratedParams(t *testing.T) {
+	t.Skip("repair this test")
```

Comment:
We can generate this params and un-skip this tests before merge

Reply:
done